variant.go

```diff
@@ -23,6 +23,18 @@ type TaskSpec struct {
 	Name     string   `json:"name" yaml:"name"`
 	Stepback bool     `json:"stepback,omitempty" yaml:"stepback,omitempty"`
 	Distro   []string `json:"distros,omitempty" yaml:"distro,omitempty"`
+	Activate *bool    `json:"activate,omitempty" yaml:"activate,omitempty"`
+}
+
+func (ts *TaskSpec) SetName(name string) *TaskSpec { ts.Name = name; return ts }
+func (ts *TaskSpec) SetStepback(shouldStepback bool) *TaskSpec {
+	ts.Stepback = shouldStepback
+	return ts
+}
+func (ts *TaskSpec) SetDistros(distros []string) *TaskSpec { ts.Distro = distros; return ts }
+func (ts *TaskSpec) SetActivate(shouldActivate bool) *TaskSpec {
```

Comment:
Activate false and activate nil have different behaviors; can this be changed to take in a boo pointerl?